.github/CONTRIBUTING.md

```diff
@@ -7,6 +7,14 @@
 Search [the issue tracker][issue tracker] to ensure that there is no open issue addressing the bug.
 If that is the case, [open a new issue][new issue]. Make sure to include a clear title, description, and as much relevant information as possible.
 
+For feature requests, refer [here](../docs/template-featureRequest.md).
+
+For bug reports, refer [here](../docs/template-bugReport.md).
+
+For new contributor introduction, refer [here](../docs/template-contributorIntro.md).
+
+For help requests, refer  [here](../docs/template-helpRequest.md).
```

Comment:
Let's make all of the above into bullet points.

And here is the fifth type, as promised:
```
For other works such as refactoring, updating documentation, or performance improvement, simply provide clear issue title and description of the work to be done.
```
Feel free to tweak the phrasing as you see fit.